File: src/main/java/dev/thornium/flatcore/api/gui/FTTextures.java
package dev.thornium.flatcore.api.gui;

import com.lowdragmc.lowdraglib.gui.texture.ResourceTexture;

public class FTTextures {
  public static final ResourceTexture PROGRESS_BAR_STONE_OREIFIER = texture("gui/progress_bar/progress_bar_stone_oreifier");

  private static ResourceTexture texture(String path) {
    return new ResourceTexture(String.format("flatcore:textures/%s.png", path));
  }
}

File: src/main/kotlin/dev/thornium/flatcore/api/gui/FTTextures.kt
package dev.thornium.flatcore.api.gui

import com.lowdragmc.lowdraglib.gui.texture.ResourceTexture

object FTTextures {
    val PROGRESS_BAR_STONE_OREIFIER = texture("gui/progress_bar/progress_bar_stone_oreifier")

    private fun texture(path: String) = ResourceTexture("flatcore:textures/$path.png")
}
